recipes/pydstool/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "0.90.2" %}
+{% set sha256 = "d2603069927f4e6f69038f90a993f810c63585b6bf428473bcd5254d940a449f" %}
+package:
+  name: pydstool
+  version: {{ version }}
+
+source:
+  fn: pydstool-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/c8/5d/9618268c502d6b38d7cfe9be7e9ee967e77c0416c1660c8dbb0368c8ce29/PyDSTool-{{ version }}.tar.gz
```

Comment:
Change this to `https://pypi.io/packages/source/P/PyDSTool/PyDSTool-{{ version }}.tar.gz`